### ISSUE TITLE
elementsOf(Iterable) for contains.inAnyOrder.atXy

### ISSUE THREAD

**Issue by robstoll** [robstoll/atrium#129]
*Platform* (jvm, jdk8, js, android): all

## Code related feature
```
val otherIterable : List<Int>
expect(listOf(1, 2)).contains.inAnyOrder.atLeast(1).elementsOf(otherIterable)

//instead of

expect(listOf(1, 2)).contains.inAnyOrder.atLeast(1).elementsOf(otherIterable.first(), *otherIterable.drop(1).toTypedArray())
```

Following the things you need to do:

*api*
- add fun `elementsOf` in apis/fluent-en_GB/atrium-fluent_en_GB-common/../iterableContainsInAnyOrderCreators.kt (see other functions there)
  - it is enough if you delegate to `values`
  - `require` that the `Iterable.hasNext`
  - add `@since 0.9.0` to the KDOC
- extend IterableContainsInAnyOrderAtLeast1ValuesAssertionsSpec (add another include, similar to  BuilderSpec)

## Your first contribution?
- Write a comment `I'll work on this` if you would like to take this issue over. 
  This way we get the chance to revise the description in case things have changed in the meantime,
  we might give you additional hints and we can assign the task to you, so that others do not start as well.
- See [Your first code contribution](https://github.com/robstoll/atrium/blob/master/.github/CONTRIBUTING.md#your-first-code-contribution) for guidelines.  
- Do not hesitate to ask questions here or to contact us via [Atrium's slack channel](https://kotlinlang.slack.com/team/U3DE1TXKP) if you need help
  ([Invite yourself](http://slack.kotlinlang.org/) in case you do not have an account yet).

**Comment by Oyedotuncaleb** [comment]
Great

**Comment by kellybrewer1967** [comment]
I don't have a computer. Just this phone

**Comment by kellybrewer1967** [comment]
I need a 64 bit codes to download on phone for fx player.

**Comment by sanatik** [comment]
I'll work on this

**Comment by robstoll** [comment]
implemented by a0b539c0
